src/jarabe/journal/listview.py

```diff
@@ -711,6 +716,17 @@ def __drag_data_get_cb(self, widget, context, selection, info, time):
         model = self.tree_view.get_model()
         model.do_drag_data_get(path, selection)
 
+    def __button_press_event_cb(self, widget, event):
+        try:
+            path = self.tree_view.get_path_at_pos(event.x, event.y)[0]
+        except:
+            return
+
+        _iter = self._model.get_iter(path)
+        _model_value = self._model.get_value(_iter, 2)
```

Comment:
Oh right! just replacing the lines
`_iter = self._model.get_iter(path)
+        _model_value = self._model.get_value(_iter, 2)`

with 

`row = self.tree_view.get_model()[path]
  _pixbuf = GdkPixbuf.Pixbuf.new_from_file(row[ListModel.COLUMN_ICON])`

would be better.